TAS_oM/TASSettings.cs

```diff
@@ -24,7 +24,7 @@
 using System.Collections.Generic;
 using BH.oM.Base;
 
-namespace BH.oM.TAS.Settings
+namespace BH.oM.Adapters.TAS
```

Comment:
Please keep `Settings` in this namespace to match XML and other Environmental adapters - `BH.oM.Adapters.TAS.Settings`